rfcs/tentative_testdriver_methods.md

```diff
@@ -0,0 +1,29 @@
+# RFC 226: Tentative testdriver methods
+
+## Summary
+Allow tentative methods to be added to testdriver.js that are not yet included in the WebDriver or WebDriver BiDi specifications.
+This should only be allowed where there is consensus between multiple browser vendors and where there is clear progress towards a specification.
+
+## Background
+As part of the [Interop 2025 Accessibility Investigation Area](https://github.com/web-platform-tests/interop-accessibility/issues/148), the intention is to extend browsers and WPT to allow additional accessibility properties to be tested by web platform tests.
+However, there are open questions regarding the shape of the API for exposing these properties, what properties should be exposed, how the tests should be written, etc.
+While the end goal is to extend the WebDriver specification with the required new endpoints, these open questions need to be answered before this is feasible.
+To answer these questions, it would be helpful if browser vendors could collaborate on a tentative but working implementation.
+
+## Details
+1. There must be publicly documented (e.g. as part of an Interop Investigation Area) consensus between at least two browser vendors and intent to work towards a specification.
+2. Any tentative methods added to testdriver.js should be prefixed with `tentative_`.
+3. Any utility methods which call tentative testdriver methods should also be prefixed with `tentative_`.
```

Comment:
I wonder if we want there to be a `tentative` object, or whether we just want to prefix everything like this?

Reply:
I wonder if having a tentative object would make it trickier to "graduate" things from tentative once the spec is final (and my thinking is that we want to incentivise finalising tentative things as soon as is feasible). With prefixing, you can basically just search/replace once the spec is final. With an object, things will need to be moved around. That's obviously not a show stopper if an object is preferred, just something I thought worth considering.